client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -445,55 +497,69 @@ const updateTheme = ( siteId, themeId ) =>
 		}
 	);
 
-const mapStateToProps = ( state, { siteId, plugins, themes } ) => {
+/**
+ * Start updating WordPress core on the specified site.
+ *
+ * @param {number} siteId  Site Id.
+ *
+ * @return {*} Stored data container for request.
+ */
+const updateCore = siteId =>
+	requestHttpData(
+		`core-update-${ siteId }`,
+		http( {
+			method: 'POST',
+			path: `/sites/${ siteId }/core/update`,
+			// No need to pass version: if it's missing, WP will be updated to latest core version.
+		} ),
+		{
+			fromApi: () => ( { version } ) => {
+				return [ [ version, true ] ];
```

Comment:
this is going to cause conflicts with the naming - `version` here will certainly be non-unique - can you add a namespace here to prevent those errors?

Reply:
in fact it doesn't even look like this information is being used. why are we storing it as the version? am I missing somewhere where we are calling `httpData( 'v2.3.0' )` or the like?